paper-checkbox-light.html

```diff
@@ -0,0 +1,263 @@
+      :host([checked]) #checkbox #checkmark {
+        -webkit-animation: checkmark-expand 140ms ease-out forwards;
+        animation: checkmark-expand 140ms ease-out forwards;
+      }
+
+      @-webkit-keyframes checkmark-expand {
+        0% {
+          -webkit-transform: scale(0, 0) rotate(45deg);
+        }
+        100% {
+          -webkit-transform: scale(1, 1) rotate(45deg);
+        }
+      }
+
+      @keyframes checkmark-expand {
+        0% {
+          transform: scale(0, 0) rotate(45deg);
+        }
+        100% {
+          transform: scale(1, 1) rotate(45deg);
+        }
+      }
+
+      :host([checked]) #checkbox {
+        background-color: var(--paper-checkbox-checked-color, var(--primary-color));
+        border-color: var(--paper-checkbox-checked-color, var(--primary-color));
+      }
+
+      #checkmark {
+        display: none;
+        position: absolute;
+        width: 36%;
+        height: 70%;
+        border-style: solid;
+        border-top: none;
+        border-left: none;
+        border-right-width: calc(2/15 * var(--calculated-paper-checkbox-size));
+        border-bottom-width: calc(2/15 * var(--calculated-paper-checkbox-size));
+        border-color: var(--paper-checkbox-checkmark-color, white);
+        -webkit-transform-origin: 97% 86%;
+        transform-origin: 97% 86%;
+        box-sizing: content-box; /* protect against page-level box-sizing */
+      }
+
+      :host([checked]) #checkmark {
+        display: block;
+      }
+
+      :host-context([dir="rtl"]) #checkmark {
+        -webkit-transform-origin: 50% 14%;
+        transform-origin: 50% 14%;
+      }
+
+      /* disabled state */
+
+      :host([disabled]) #checkbox {
+        opacity: 0.5;
+        border-color: var(--paper-checkbox-unchecked-color, var(--primary-text-color));
+      }
+
+      :host([disabled][checked]) #checkbox {
+        background-color: var(--paper-checkbox-unchecked-color, var(--primary-text-color));
+        opacity: 0.5;
+      }
+    </style>
+
+    <div id="checkbox">
+      <div id="checkmark"></div>
+    </div>
+  </template>
+
+  <script>
+    Polymer({
+      is: 'paper-checkbox-light',
+
+      behaviors: [
+        Polymer.PaperRippleBehavior
+      ],
+
+      hostAttributes: {
+        role: 'checkbox',
+        'aria-checked': false,
+        tabindex: 0
+      },
+
+      properties: {
+        checked: {
+          type: Boolean,
+          notify: true,
+          reflectToAttribute: true,
+          observer: '_checkedChanged'
+        },
+
+        disabled: {
+          type: Boolean,
+          reflectToAttribute: true,
+          observer: '_disabledChanged'
+        },
+      },
+
+      listeners: {
+        'tap': '_onTap',
+        'keypress': '_onKeypress',
+        'down': '_rippleDown',
+        'up': '_rippleUp',
+        'focus': '_rippleDown',
+        'blur': '_rippleUp'
+      },
+
+      attached: function() {
+        this.setAttribute('aria-checked', !!this.checked);
```

Comment:
I think for the falsy case it needs to be `this.removeAttribute('aria-checked')`. Same for disabled.